### ISSUE TITLE
Request: Android TV support

### ISSUE THREAD

**Issue by snoozysoft** [AbdullahSako/ChikiChikiTube-Android#6]
Just tried sideloading this onto a Fire TV Stick and whilst it loaded up fine and I could interact with the menu buttons on the bottom, I wasn't able to actually click on anything else. It'd be super nice to have this app available for Android based TV sticks and adjacent things since loading the site up in the browser will just lead to mixed to negative results. Plus the convenience of it would be huge. Many thanks for your work so far, its otherwise been working a treat for me!

**Comment by double-u-a** [comment]
I can confirm Samsung TV Internet browser runs the website pretty much perfectly, and on Android you can install up to date browsers like Chrome and Firefox pretty easily, when you say mixed to negative results is there a specific TV browser you are having trouble on?

**Comment by snoozysoft** [comment]
The Fire TV Stick is rather locked down and you're kinda stuck with their own Amazon Silk browser. Using that to open the site results in it not only rendering incorrectly but also being incredibly slow. Not to mention it is very arduous to type in the URL with their non-qwerty keyboard using the remote. Having the app properly work with it would be massively more convenient and function better too.

**Comment by double-u-a** [comment]
I suggest you rename the issue to Android TV support, probably best to focus on that first as it's not locked down like Fire TV.

**Comment by AbdullahSako** [comment]
yeah i am planning on making an app for android tv
alot of people have been asking for one

**Comment by snoozysoft** [comment]
Hey, decided to check in on this and saw that you seem to have some preliminary tv support set up; if you wanna push out a test apk or something of it i'd be happy to test it for you.

**Comment by AbdullahSako** [comment]
I am trying to push it to play store but google wont let me, the first few times they told me to change a few things, after making those changes they still denied tv support and did not elaborate, i did make an appeal but they did not answer, i will try a few things but its just that ive been busy with work.

**Comment by snoozysoft** [comment]
It's cool, dont worry about it, there's no rush. If you get time, add the apk to the github releases page and I'll just sideload it. I don't have access to the play store from my TV stick anyway.

**Comment by double-u-a** [comment]
Agreed, I think we should just go with sideloading an apk as the official recommended way for now and have it out there so people can test it.

**Comment by AbdullahSako** [comment]
done! its now on github releases and should be on fdroid in a few days
